database/engine/global_uuid_map/global_uuid_map.c

```diff
@@ -244,10 +244,12 @@ int find_or_generate_guid(void *object, uuid_t *uuid, GUID_TYPE object_type, int
         uv_rwlock_wrunlock(&global_lock);
         return rc;
     }
-    //uv_rwlock_wrunlock(&global_lock);
+    else {
```

Comment:
You don't really need the `else` here because the `if` above it returns :)

Reply:
> You don't really need the `else` here because the `if` above it returns :)

How do you DO that!! :smile:

Reply:
No shame!!!

Reply:
lol